app/assets/templates/sidebar.html.erb

```diff
@@ -35,6 +35,19 @@
   
     <div class="sidebar-pane" id="profile">
       <h1 class="sidebar-header">Profile<span class="sidebar-close"><i class="fa fa-caret-left"></i></span></h1>
+      <div ng-if="!$ctrl.user.guest">
```

Comment:
to be changed

Reply:
all guest users can see files here? is that what this does?

Reply:
the opposite, so if the user is _**not**_ a guest, they can see the files along with other profile information. below, i'm making it so that if the user _**is**_ a guest, they will only see a link to login